### PR TITLE
feat: naming conventions, doxygen, includes in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by theophile-jr**
We must improve our CONTRIBUTING.md
Please feel free to give your opinions.

### CODE REVIEW

**Review by Thomaltarix — APPROVED** [review]

**Review by Njord201 — APPROVED** [review]

**Review by mathieurobert1 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,3 +25,34 @@ To maintain an organized repository, we have naming conventions for branches. He
...
+
+- In C : snake_case (Epitech)
+- In C++ : 
+    - variables = camelCase
+    - classes = PascalCase
+    - functions = camelCase
+    - constants = ALL_CAPS
+    - macros = ALL_CAPS
+    - namespaces = PascalCase
+    - enums = camelCase
```
Comment (Marius-P1) [inline CONTRIBUTING.md:41]: Here is about the name of the enum of the content ? If it's the content i think it need to be ALL_CAPS

**Review by Marius-P1 — APPROVED** [review]
Good for me just a small question :ok_hand:

File: CONTRIBUTING.md (hunk 1 above)
Comment (mathieurobert1) [reply CONTRIBUTING.md:41]: No it's the name of the enums, not the content.

Comment (mathieurobert1) [reply CONTRIBUTING.md:41]: But @theophile-jr can add the content.

**Review by AxelF44 — APPROVED** [review]

**Pull request merged**